packages/adapter-next/src/hooks/slice-template-library-read.ts

```diff
@@ -0,0 +1,46 @@
+import fs from "node:fs";
```

Comment:
This file should be named `sliceTemplateLibrary-read` to match the other hook file names.

`slice-template-library:read` => `sliceTemplateLibrary-read`
`custom-type:read` => `customType-read`
`slice:read` => `slice-read`